src/content/products/sparseml.mdx

```diff
@@ -152,18 +145,21 @@ SparseML enables you to create a sparse model with **Sparse Transfer Learning**
 To enable flexibility, ease of use, and repeatability, each piece of functionality is accomplished via recipes.
   The recipes encode the instructions needed for modifying the model and/or training process as a list of modifiers.
   Example modifiers can be anything from setting the learning rate for the optimizer to gradual magnitude pruning.
-  The files are written in [YAML](https://yaml.org/) and stored in YAML or [markdown](https://www.markdownguide.org/) files using [YAML front matter.](https://assemble.io/docs/YAML-front-matter.html) The rest of the SparseML system is coded to parse the recipes into a native format for the desired framework and apply the modifications to the model and training pipeline.
+  The files are written in [YAML](https://yaml.org/) and stored in YAML or [Markdown](https://www.markdownguide.org/) files using [YAML front matter.](https://assemble.io/docs/YAML-front-matter.html) The rest of the SparseML system is coded to parse the recipes into a native format for the desired framework and apply the modifications to the model and training pipeline.
 
-To give a sense of the flavor of what recipes encode, some examples are below:
+To give a sense of the flavor of what recipes encode, some examples are:
 - Recipes for **Sparse Transfer Learning** usually include the `!ConstantPruningModifier`, which instructs SparseML to maintian the starting level of sparsity while fine-tuning.
 
-- Recipes for **Sparsification from Scratch** usually include the `!GMPruningModifier`, which instructs SparseML to iteratively prune the layers of the model to certain levels (e.g. 80%) over which epochs.
+- Recipes for **Sparsification from Scratch** usually include the `!GMPruningModifier`, which instructs SparseML to iteratively prune the layers of the model to certain levels (e.g., 80%) over which epochs.
 
 Recipes are then integrated into deep learning training workflows in one of two ways:
 
-#### For Supported Use Cases: CLI
+- For supported use cases: CLI
+- For custom use cases / supported use cases: Python integration
+
+#### Supported Use Cases: CLI
 
-SparseML provides command line scripts that accept recipes as arguments and perform Sparse Transfer Learning and Sparsification from Scratch. We highly
+SparseML provides command line scripts that accept recipes as arguments and perform sparse transfer learning and sparsification from scratch. We highly
 reccomended using the command line scripts. Appending `--help` to the commands demonstrates the full list of arguments.
 
 For example, the following command kicks off **Sparse Transfer Learning** from pre-sparsified YOLOv5 onto the VOC dataset using the pre-made recipes in the SparseZoo:
```

Comment:
Is "Sparse Transfer Learning" is a specific formal use case requiring case sensitive grammar? It feels like we're generally covering two different scenarios. cc @Beth-Kosis and @rsnm2

Reply:
I am trying to brand those terms, can discuss more at the PMM meeting tomorrow